crates/transaction-emitter-lib/src/cluster.rs

```diff
@@ -96,9 +96,10 @@ impl Cluster {
             let state_timestamp = state.timestamp_usecs / 1000000;
             if state.chain_id != chain_id.id() {
                 warn!(
-                    "Excluding client {} running wrong chain {}",
+                    "Excluding client {} running wrong chain {}, instead of {}",
```

Comment:
Just curious, when does this happen?

Reply:
If devnet gets upgraded, and I forget to update the value. 

or hit devnet endpoints with command for testnet